forte/data/data_store.py

```diff
@@ -324,7 +327,11 @@ def _new_annotation(self, type_name: str, begin: int, end: int) -> List:
         return entry
 
     def _new_link(
-        self, type_name: str, parent_tid: int, child_tid: int
+        self,
+        type_name: str,
+        parent_tid: int,
+        child_tid: int,
+        tid: Optional[int] = None,
```

Comment:
add these new parameters to the docstring

Reply:
Added.